### PR TITLE
Use service call for collision-object-publisher

### PR DESCRIPTION
**Pull request by knorth55**
Improve version of #317 (thanks to @eisoku9618)

Use `ApplyPlanningScene` service to operate `collision_objects` and `attached_collision_objects`.
Currently, `publish and sleep` method is implemented, but the old-style method is not recommended. (https://github.com/ros-planning/moveit_msgs/pull/40#issuecomment-335734542)
We use `service-call` method instead.

### CODE REVIEW

**Comment by eisoku9618** [comment]
Thank you! You can squash my commit and your first commit if you like.

File: pr2eus_moveit/euslisp/collision-object-publisher.l [hunk 1]
```diff
@@ -174,52 +178,62 @@
...
-         (send msg :object :operation moveit_msgs::CollisionObject::*REMOVE*)
-         (ros::publish attached-topicname msg)
-         ;; According to moveit tutorials, we need to put sleep after topic publishing.
-         ;; http://docs.ros.org/indigo/api/pr2_moveit_tutorials/html/planning/src/doc/planning_scene_ros_api_tutorial.html
-         ;; adding unix::usleep is not smart solution, but I couldn't find out better solution.
-         ;; https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/315#issuecomment-315015718
-         (unix::usleep (* 500 1000)))
+       (send req :scene :robot_state :attached_collision_objects
+             (mapcar #'(lambda (msg)
+                         (progn
```
Comment (YoheiKakiuchi) [inline pr2eus_moveit/euslisp/collision-object-publisher.l]: progn is not required inside lamda

Comment (knorth55) [reply pr2eus_moveit/euslisp/collision-object-publisher.l]: thank you so much! I updated.

**Comment by knorth55** [comment]
I checked with baxter and it works with no problem.

File: pr2eus_moveit/euslisp/collision-object-publisher.l (hunk 1 above)
Comment (YoheiKakiuchi) [reply pr2eus_moveit/euslisp/collision-object-publisher.l]: +1

**Pull request merged**